src/alt/store/index.js

```diff
@@ -53,7 +53,13 @@ export function createStoreConfig(globalConfig, StoreModel) {
         return state
       }
     },
-    setState: fn.assign
+    setState(currentState, nextState) {
+      if (Object.prototype.toString.call(nextState) === '[object Object]') {
```

Comment:
yeah fn.isPlainObject sounds like a good idea. Not sure if I should merge #468 or not. It's a bit dirty to add a property to alt stores and check for them.